fapolicy_analyzer/ui/actions.py

```diff
@@ -41,6 +42,29 @@
 RECEIVED_GROUPS = "RECEIVED_GROUPS"
 ERROR_GROUPS = "ERROR_GROUPS"
 
+INIT_DAEMON = "INIT_DAEMON"
+DAEMON_INITIALIZED = "DAEMON_INITIALIZED"
+
+REQUEST_DAEMON_START = "REQUEST_DAEMON_START"
+RECEIVED_DAEMON_START = "RECEIVED_DAEMON_START"
+ERROR_DAEMON_START = "ERROR_DAEMON_START"
+
+REQUEST_DAEMON_STOP = "REQUEST_DAEMON_STOP"
+RECEIVED_DAEMON_STOP = "RECEIVED_DAEMON_STOP"
+ERROR_DAEMON_STOP = "ERROR_DAEMON_STOP"
+
+REQUEST_DAEMON_STATUS = "REQUEST_DAEMON_STATUS"
+RECEIVED_DAEMON_STATUS = "RECEIVED_DAEMON_STATUS"
+ERROR_DAEMON_STATUS = "ERROR_DAEMON_STATUS"
+
+REQUEST_DAEMON_STATUS_UPDATE = "REQUEST_DAEMON_STATUS_UPDATE"
```

Comment:
What is the difference between `REQUEST_DAEMON_STATUS` and `REQUEST_DAEMON_STATUS_UPDATE` actions?

Reply:
To me it looks like you can consolidate the `STATUS` and `STATUS_UPDATE` sets of actions into a single set.  Its ok for multiple places to use the same actions, so you can dispatch the same actions from the `main_window` menu item and from your checker background thread.

Reply:
We can remove REQUEST_DAEMON_STATUS which is a distinct request invoked from the System menu. My original idea was acquire other fapolicyd info e.g. version, build date, etc, and run status and present it in a dialog when explicitly requested manually via the System menu. The REQUEST_DAEMON_STATUS_UPDATE was associated with the timed polling loop in the feature code. I'll revisit that.

Reply:
Removed REQUEST_DAEMON_STATUS and associated variants, RECEIVED, and ERROR.